lib/oper.g

```diff
@@ -1734,20 +1734,44 @@ end );
 ##  global variables (see <C>variable.g</C>) because of the completion
 ##  mechanism.
 ##
+
+BIND_GLOBAL( "UTILS_FUNCTION_NAMES",                    ## for Utils 
+    ShareSpecialObj([], "UTILS_FUNCTION_NAMES") );      ## for Utils 
+BIND_GLOBAL( "UTILS_FUNCTION_OPERS",                    ## for Utils 
+    ShareSpecialObj([], "UTILS_FUNCTION_OPERS") );      ## for Utils 
+BIND_GLOBAL( "UTILS_FUNCTION_COUNT",                    ## for Utils 
+    ShareSpecialObj([], "UTILS_FUNCTION_COUNT") );      ## for Utils 
+
 BIND_GLOBAL( "GLOBAL_FUNCTION_NAMES", ShareSpecialObj([], "GLOBAL_FUNCTION_NAMES") );
```

Comment:
I dislike that these new globals are complelty undocumented. If somebody looks at that code in a couple years, they will have no idea what it is about, and would be forced to dig through the version history of the repository to discover its meaning.

So, I really think there should be a block comment there, in the code, which explains what this is all about.